datasets/goes/goes-glm/collection/template.json

```diff
@@ -0,0 +1,334 @@
+            "type": "application/x-parquet",
+            "title": "Processed GeoParquet file for groups",
+            "table:primary_geometry": "geometry",
+            "table:columns": [
+                {
+                    "name": "geometry",
+                    "type": "point"
+                },
+                {
+                    "name": "id",
+                    "type": "int32",
+                    "description": "product-unique lightning group identifier"
+                },
+                {
+                    "name": "time",
+                    "type": "datetime"
+                },
+                {
+                    "name": "time_offset",
+                    "type": "int16",
+                    "description": "GLM L2+ Lightning Detection: mean time of group's constituent events' times of occurrence"
+                },
+                {
+                    "name": "frame_time",
+                    "type": "datetime"
+                },
+                {
+                    "name": "frame_time_offset",
+                    "type": "int16",
+                    "description": "GLM L2+ Lightning Detection: mean time of group's constituent events' times of occurrence"
+                },
+                {
+                    "name": "area",
+                    "type": "int16",
+                    "description": "GLM L2+ Lightning Detection: group area coverage (pixels containing at least one constituent event only)",
+                    "unit": "m2"
+                },
+                {
+                    "name": "energy",
+                    "type": "int16",
+                    "description": "GLM L2+ Lightning Detection: group radiant energy",
+                    "unit": "J"
+                },
+                {
+                    "name": "quality_flag",
+                    "type": "int16",
+                    "description": "GLM L2+ Lightning Detection: group data quality flags"
+                },
+                {
+                    "name": "parent_flash_id",
+                    "type": "int16",
+                    "description": "product-unique lightning flash identifier for one or more groups"
+                }
+            ],
+            "roles": [
+                "data"
+            ]
+        },
+        "netcdf": {
+            "type": "application/netcdf",
+            "title": "Original NetCDF4 file",
+            "roles": [
+                "data"
+            ]
+        }
+    },
+    "summaries": {
+        "mission": [
+            "GOES"
+        ],
+        "constellation": [
+            "GOES"
+        ],
+        "platform": [
+            "GOES-16",
+            "GOES-17"
+        ],
+        "instruments": [
+            "FM1",
+            "FM2"
+        ],
+        "gsd": [
+            8000
+        ],
+        "processing:level": [
+            "L2"
+        ],
+        "goes:orbital-slot": [
+            "West",
+            "East",
+            "Test"
```

Comment:
Did we decide whether there were actually any `Test` assets we would make items for?

Reply:
Yeah, there are assets in the Test orbital slot. We were concerned about assets during the drift from the Test to East/West slot, but we found that there is no data (thankfully) for the time period where the satellites were drifting.